classes/opponentPhaseRolling.lua

```diff
@@ -0,0 +1,74 @@
+-- The opponent phase with normal gravity
+
+local opponentPhaseRolling = {}
+opponentPhaseRolling.__index = opponentPhaseRolling
+
+
+--------------------
+-- MAIN CALLBACKS --
+--------------------
+
+function opponentPhaseRolling.new()
+	local self = classes.opponentBase.new()
+	setmetatable(self, opponentPhaseRolling)
+
+	-- Constants
+	self.MOVE_SPEED = 200
+	self.BULLET_COOLDOWN_LENGTH = 1
+	GRAVITY = 250
```

Comment:
Move this line to above the "-- MAIN CALLBACKS --" comment line, make it a local variable. Maybe make it something like `local OPPONENT_GRAVITY = ...`